packages/core/src/lib/light_push/index.ts

```diff
@@ -42,8 +43,12 @@ type PreparePushMessageResult =
  * Implements the [Waku v2 Light Push protocol](https://rfc.vac.dev/spec/19/).
  */
 class LightPush extends BaseProtocol implements ILightPush {
-  constructor(libp2p: Libp2p, options?: ProtocolCreateOptions) {
-    super(LightPushCodec, libp2p.components, log, options);
+  constructor(
+    libp2p: Libp2p,
+    pubsubTopics: PubsubTopic[],
+    options?: ProtocolCreateOptions
+  ) {
```

Comment:
same comment as `filter` to move `pubsubTopics` within `ProtocolCreateOptions` as required

Reply:
updated in latest commit